products/index.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../styles/main.css" />
+    <link
+      rel="stylesheet"
+      href="https://deploy-preview-8--keen-meninsky-a05bc6.netlify.app/home.css"
+    />
+    <link
+      rel="shortcut icon"
+      href="../assets/favicon.ico"
+      type="image/x-icon"
+    />
+    <link
+      rel="stylesheet"
+      href="https://keen-meninsky-a05bc6.netlify.app/home.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://deploy-preview-8--keen-meninsky-a05bc6.netlify.app/styles/utility.css"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <title>E-commerce</title>
+  </head>
+  <body>
+    <nav class="navbar">
+      <a href="/" class="navbar-title">Neo Cart</a>
+      <div class="navbar-search-input-container">
+        <div class="input-lefticon">
+          <img src="./assets/images/search-icon.svg" alt="" />
+        </div>
+        <input class="navbar-search-input" type="text" placeholder="Search" />
+      </div>
+      <div class="navbar-right-options">
+        <a href="/login">
+          <button>Login</button>
+        </a>
+        <div>
+          <a href="/wishlist">
+            <img height="24px" src="../assets/images/Wishlist.svg" alt="" />
+          </a>
+        </div>
+        <div>
+          <a href="/cart" class="flex-row items-center">
+            <img height="24px" src="../assets/images/Cart-Icon.svg" alt="" />
```

Comment:
don't style elements in the HTML, try to write all the styles in a CSS file only